File: python/Day_1/Question_11.py
a = int(input('a: '))
b = int(input('b: '))
c = int(input('c: '))

def compare(a, b, c):
    if(a > b and a > c):
        return f"{a} is greater than {b} and {c}"
    elif(b > a and b > c):
        return f"{b} is greater than {a} and {c}"
    elif(c > a and c > b):
        return f"{c} is greater than {a} and {b}"
    else:
        return "There is a tie among the values."

result = compare(a, b, c)
print(result)

File: javascript/Day_1/Question_11.js
const a = 10;
const b = 13;
const c = 15;

const check = (a, b, c) => {
  if (a > b && a > c) {
    return `${a} is greater than the ${b} and ${c}`;
  } else if (b > a && b > c) {
    return `${b} is greater than the ${a} and ${c}`;
  } else if (c > b && c > a) {
    return `${c} is greater than the ${b} and ${a}`;
  } else {
    return `${a}, ${b} and ${c} are equal`;
  }
};

const result = check(a, b, c);

console.log(result);
